### ISSUE TITLE
Add alt text to images in markdown files

### ISSUE THREAD

**Issue by Frijol** [edgi-govdata-archiving/EEW-Website#24]
I've added images in the markdown files that host the content for the site (the ones in content/pages). For accessibility reasons, each image should have some short descriptive alt-text (the text you see when you mouse over an image, and that screen readers will read out).

The format for Markdown images is `![alt text](link to the image)`

@lesliealanis or @calderEDJ does one of you want to do this? It's a small task that would get you started editing files in the website

See if you can figure out what to edit by reading my docs here: https://github.com/edgi-govdata-archiving/EEW-Website#where-to-edit-content but also feel free to hit me up for help

**Comment by lesliealanis** [comment]
#34

**Comment by lesliealanis** [comment]
@Frijol I ended up solving it on issue #34
